extensions/2.0/Vendor/EXT_feature_metadata/schema/featureIdTexture.schema.json

```diff
@@ -2,15 +2,19 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "title": "Feature ID Texture",
   "type": "object",
+  "allOf": [ { "$ref": "textureInfo.schema.json" } ],
   "description": "An object describing a texture used for storing per-texel feature IDs.",
   "properties": {
+    "index": { },
+    "texCoord": { },
     "featureTable": {
       "type": "string",
       "description": "The ID of the feature table in the model's root `EXT_feature_metadata.featureTables` dictionary."
     },
-    "featureIds": {
-      "allOf": [ { "$ref": "textureAccessor.schema.json" } ],
-      "description": "A description of the texture and channel to use for feature IDs. The `channels` property must have a single channel. Furthermore, feature IDs must be whole numbers in the range `[0, count - 1]` (inclusive), where `count` is the total number of features in the feature table. Texel values must be read as integers. Texture filtering should be disabled when fetching feature IDs."
+    "channels": {
+      "type": "string",
+      "pattern": "^[rgba]$",
+      "description": "Single channel (`\"r\"`, `\"g\"`, `\"b\"`, or `\"a\"`) storing per-texel feature IDs."
```

Comment:
I see you removed the description of feature IDs. Was that intentional because the same thing happens in the main text?

Reply:
Yeah, I just trimmed it down to avoid duplicating text that might need more revisions. Happy to restore the detail here (either now, or closer to final draft) if we'd prefer.